docs/src/components/document-renderer.tsx

```diff
@@ -111,4 +113,18 @@ const componentBlockRenderers: InferRenderersForComponentBlocks<
       </div>
     );
   },
+  'cloud-image': ({ href: src, alt, sizes, height, width }) => {
+    const imgMaxWidthPx = `${parseInt(CONTENT_MAX_WIDTH_DESKTOP) * 16}`;
+
+    return (
+      <img
+        alt={alt || ''}
+        src={src}
+        height={height}
+        width={width || imgMaxWidthPx}
+        srcSet={getDefaultSrcSet({ src })}
```

Comment:
Should this guy have a `srcSet || getDefaultSrcSet({ src })` potentially? Or maybe the plan is to not expose this field in the future really?

Reply:
Yes, it should!